micb_soa.m

```diff
@@ -240,20 +241,49 @@
             timesUp = 1;
         end
     end
-    
+    %%% Create a 4X8 double 
+    non_targets = centredRects(1:8);
+    %%% Reduce to a 4X7 by removing the target
+    non_targets(:,target) = [];
+    incorrectRect = centeredRects(:,non_targets);
+    incorrectRect = incorrectRect;
     correctRect = centeredRects(:,target);
     correctRect = correctRect';
     
+    %%% KeySet for all dictionaries

```

Comment:
nice to see you use the dictionary structure again, can you comment this a bit better with what each step is doing , this new keys and maps is confusing what the purpose of this is, so state that clearly so people unfamiliar with containers.Map could get it